README.md

```diff
@@ -129,7 +137,7 @@ You don't have to ever use `eject`. The curated feature set is suitable for smal
 
 ### Deployment
 
```

Comment:
## 🟡 Good job so far 🥳🎉👏 
#### Your app is complete actually but you forgot some things in readme 😁
- [ ] Please fill the *Acknowledgements* section in readme with the link to the author of the original design 🙂
- [ ] Please also kindly remove the requirements from the readme file. **TIP**: You can put it to the pull request description because usually other people see the result and we reviewers need the requirement to check your code ok 😉😅

Reply:
There is an acknowledgment in the description (design section.) I can move it in the Acknowledgments section.
I always put the requirements in the README file. No one else said something about them. If this is what I have to do I can remove them and put them in the PR.

Reply:
Yes, it is not a good idea to put the requirements in the readme file. Without it, it will be easy to read or understand the project from the readme. Also with some privacy reasons, because requirements are for microverse students but anyone can see your project from GitHub 👍✅